saleor/graphql/notifications/mutations/external_notification_trigger.py

```diff
@@ -0,0 +1,95 @@
+import graphene
+
+from ....core.exceptions import PermissionDenied
+from ....core.notification.mutation_handler import (
+    get_external_notification_payload,
+    send_notification,
+)
+from ....core.notification.validation import (
+    validate_and_get_channel,
+    validate_and_get_external_event_type,
+    validate_and_get_payload_params,
+    validate_ids_and_get_model_type_and_pks,
+)
+from ....graphql.core.types.common import ExternalNotificationError
+from ...core.mutations import BaseMutation
+
+
+class ExternalNotificationTriggerInput(graphene.InputObjectType):
+    ids = graphene.List(
+        graphene.ID,
+        required=True,
+        description=(
+            "The list of customers or orders node IDs that will be serialized and "
+            "included in the notification payload."
+        ),
+    )
+    extra_payload = graphene.JSONString(
+        description=(
+            "Additional payload that will be merged with "
+            "the one based on the bussines object ID."
+        )
+    )
+    external_event_type = graphene.String(
+        required=True,
+        description=(
+            "External event type. In case of invalid type, "
+            "Saleor will consider the content"
+            "of that field like a ID of dynamic template."
+        ),
+    )
+    channel = graphene.String(
```

Comment:
I would consider if this place is right. Your mutation has input: `input` and `plugin_id`. Plugin_id determines the configuration of running this mutation. `Chanel` does the same.  For me, it should be in the same place as `plugin_id`